app/models/tag.rb

```diff
@@ -20,15 +20,18 @@ class Tag < ActsAsTaggableOn::Tag
             format: /\A#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})\z/, allow_nil: true
   validates :category, inclusion: { in: ALLOWED_CATEGORIES }
 
-  validate :validate_alias
-  validate :validate_name
+  validate :validate_alias_for, if: proc { alias_for.present? }
+  validate :validate_name, if: proc { name.present? }
```

Comment:
Btw, could be `if: :name?` and `if: :alias_for?`

Reply:
Woah, neat tip! Didn't know, let me try :)

Reply:
Works great! Signed, sealed, delivered :D